### PR TITLE
Added a tip about hashing the result of nextBytes()

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | #4328 |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
This little contribution needs some review. Thanks!

File: components/security/secure_tools.rst [hunk 1]
```diff
@@ -54,7 +54,14 @@ to work correctly. Just pass a file name to enable it::
...
-    instance directly from the container: its name is ``security.secure_random``.
+    If you're using the Symfony Framework, you can get a secure random number
+    generator via the ``security.secure_random`` service.
+
+.. tip::
+
+    The ``nextBytes()`` method returns a binary string which may contain the
+    ``\0`` character. If you store this value in a database or include it as
+    part of the URL, make sure to hash the value returned by ``nextBytes()``
+    (to do that, you can use a simple ``md5()`` PHP function).
```
Comment (xabbuh) [inline components/security/secure_tools.rst]: Not sure if we should limit the tip to these two scenarios. There could be lots of third-party systems you need so send data to which may or may not be capable of handling strings containing the null byte. Could we/should we write this more generic?

Comment (javiereguiluz) [inline components/security/secure_tools.rst]: I agree. I've reworded this tip. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Awesome! Minor tweaks at sha: 6a34332c20a14b86547e9e7e177c14d0fe7ebed5

**Pull request merged**